tests/testthat/test-betareg.R

```diff
@@ -32,6 +32,24 @@ test_that("tidy.betareg", {
   )
 
   check_dims(td1, 12, 8)
+  
+  #test passing ellipsis to summary
+  data_test <- structure(
+    list(p = c(0.4238, 0.8248, 0.5927, 0.3208, 0.8317, 
+               0.7314, 0.4624, 0.5224, 0.3528, 0.7739, 0.1264, 0.4516, 0.4306, 
+               0.5243, 0.5117, 0.149, 0.3342, 0.5069, 0.7101, 0.8019, 0.7569, 
+               0.9096, 0.9013, 0.5403, 0.2264, 0.5775, 0.7366, 0.4086, 0.575, 
+               0.6623, 0.7758, 0.1517, 0.7587, 0.3247, 0.7463, 0.6325), 
+         w = structure(c(1L, 2L, 9L, 4L, 3L, 5L, 1L, 9L, 4L, 3L, 5L, 2L, 4L, 5L, 1L, 2L, 9L, 
+                         4L, 3L, 5L, 8L, 7L, 6L, 1L, 2L, 9L, 3L, 5L, 8L, 7L, 1L, 2L, 9L, 
+                         4L, 3L, 5L), 
+                       levels = c("1", "2", "3", "4", "5", "6", "7", "8", "9"), 
+                       class = "factor")), 
+    row.names = c(NA, -36L), class = c("tbl_df", "tbl", "data.frame")
+  )
+  m <- betareg::betareg(p ~ w, data = data_test)
+  expect_warning(tidy(m))
+  expect_silent(tidy(m, type = "sweighted"))
```

Comment:
Rather than inlining data and making a new fit here, could you just use `fit1` from above, pass an argument through `tidy()`'s `...` that induces a change in output, and `expect_*()` that that output changes?

Reply:
The point of changing the type of residual is that the default one (`sweighted2`) may produce NaN, as explained in detail in https://github.com/tidymodels/broom/issues/1198. 
This inline data is the minimal example I could find to reproduce this behavior. Isn't that right?